src/Psalm/ErrorBaseline.php

```diff
@@ -198,16 +200,11 @@ public static function update(
      */
     private static function countIssueTypesByFile(array $issues): array
     {
-        $normalized_data = [];
-
-        foreach ($issues as $file_issues) {
-            foreach ($file_issues as $issue_data) {
-                $normalized_data[] = $issue_data;
-            }
+        if (0 === count($issues)) {
```

Comment:
`[] === $issues` is "much" faster than `count`

Reply:
updated in 947765c

Reply:
good spot. strangely that i knew that and already do it in all my code, but didn't here! thanks!